src/tests/validation/pka_test_validation.c

```diff
@@ -897,6 +899,58 @@ static void ShiftTest(thread_args_t *args,
     TestFailed(args, __func__, pki_fcn_name, &operand, 1, rc, result, correct);
 }
 
+static void DhTest(thread_args_t *args,
+                   uint32_t       value_idx,
+                   uint32_t       exponent_idx,
+                   uint32_t       modulus_idx,
+                   uint32_t       correct_idx)
+{
+    pka_operand_t *value, *exponent, *modulus, *correct, *result, *inputs[3];

```

Comment:
bad indent. Has to be consistent with the rest of the code

Reply:
Thanks @kblaiech.